File: 백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    
	    int N = sc.nextInt();
	    int min = sc.nextInt();
	    int max = min;
	    
	    for (int i = 1; i < N; i++) {
	        int tmp = sc.nextInt();
	        
	        if (max < tmp) max = tmp;
	        if (min > tmp) min = tmp;
	    }
	    
		System.out.printf("%d %d", min, max);
	}
}

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.c
#include <stdio.h>

int main()
{
    int N, min, max, tmp;
    scanf("%d", &N);
    scanf("%d", &min);
    max = min;
    
    for (int i = 1; i < N; i++) {
        scanf("%d", &tmp);
        
        if (max < tmp) max = tmp;
        if (min > tmp) min = tmp;
    }
    
    printf("%d %d", min, max);
}